### PR TITLE
More minor cleanups (no functional change)

### PR DESCRIPTION
**Pull request by lyricnz**

### CODE REVIEW

File: cointop/config.go [hunk 1]
```diff
@@ -227,9 +227,9 @@ func (ct *Cointop) ConfigToToml() ([]byte, error) {
 		if !ok || entry.Coin == "" {
 			continue
 		}
-		var amount string = strconv.FormatFloat(entry.Holdings, 'f', -1, 64)
-		var coinName string = entry.Coin
-		var tuple []string = []string{coinName, amount}
+		var amount = strconv.FormatFloat(entry.Holdings, 'f', -1, 64)
```
Comment (miguelmota) [inline cointop/config.go:230]: suggestion: use shorthand

```go
amount := ...
coinName := ...
tuple := ...
```

File: cmd/commands/server.go [hunk 2]
```diff
@@ -15,14 +15,14 @@ import (
 // ServerCmd ...
 func ServerCmd() *cobra.Command {
 	var port uint = 22
-	var address string = "0.0.0.0"
+	var address = "0.0.0.0"
```
Comment (miguelmota) [inline cmd/commands/server.go:18]: suggestion: use shorthand for all variables

example

```go
idleTimeout := uint(0)
hostKeyFile := cssh.DefaultHostKeyFile
```

File: cointop/config.go (hunk 1 above)
Comment (lyricnz) [reply cointop/config.go:230]: Seems like lots of places we could do that? Happy to replace 'em all?

File: cmd/commands/server.go (hunk 2 above)
Comment (lyricnz) [reply cmd/commands/server.go:18]: what do you think about `var port uint = 22` vs `port := uint(22)`?

Comment (lyricnz) [reply cmd/commands/server.go]: Have left those as-is for now, so we can merge this.  "cleanup" is never-ending process :)

Comment (miguelmota) [reply cmd/commands/server.go]: I think type inferred declarations are preferred in general. There's a lot of places in the code that can be updated to use the shorthand for sure

Comment (lyricnz) [reply cmd/commands/server.go]: Not as many as I thought. Only found 9 further matches for `r/var \w+ \w+ =/` mostly of the uint pattern above. Updating now to uint(x) version.

Screen Shot 2021-10-03 at 4 42 50 pm

Comment (lyricnz) [reply cmd/commands/server.go]: Pushed a change for this - hope I understand correctly!

File: cmd/commands/server.go [hunk 3]
```diff
@@ -14,15 +14,15 @@ import (
...
-	var executableBinary string = "cointop"
-	var hostKeyFile string = cssh.DefaultHostKeyFile
-	var userConfigType string = cssh.UserConfigTypePublicKey
-	var colorsDir string = os.Getenv("COINTOP_COLORS_DIR")
+	var port = uint(22)
+	address := "0.0.0.0"
+	idleTimeout := uint(0)
+	maxTimeout := uint(0)
+	maxSessions := uint(0)
+	var executableBinary = "cointop"
```
Comment (miguelmota) [inline cmd/commands/server.go:22]: shorthand can be used here
```
var executableBinary = "cointop"
```

**Review by miguelmota — APPROVED** [review]

**Pull request merged**